src/app/pages/login/login.component.ts

```diff
@@ -28,6 +30,9 @@ export class LoginComponent implements OnInit, OnDestroy {
     this.waiting = false;
     this.success = login.isAuthenticated;
     if (this.success) {
+      const token = await firstValueFrom(this.security.getAccessToken());
+        this.incidentService.configuration.withCredentials = true;
+        this.incidentService.defaultHeaders = this.incidentService.defaultHeaders.set("Authorization", `Bearer ${token}`);
```

Comment:
I think this is the "global login"?

Then `IncidentService` isn't the only API client service. What about `component`, `phase`, etc.?

Reply:
Indeed, the login component is accessed whenever a user logs in, so it would be the appropriate place for doing this.

And yes, currently, only the `IncidentService` is getting the necessary header, simply because we currently do not support any operations on any of the other services that would require write access. If one where to add this, they'd have to add setting the appropriate headers.